packages/loopring_v3/contracts/aux/access/LoopringIOExchangeOwner.sol

```diff
@@ -90,7 +92,10 @@ contract LoopringIOExchangeOwner is SelectorBasedAccessManager
         }
 
         // Finally submit the blocks
-        IExchangeV3(target).submitBlocks(blocks);
+        IExchangeV3(target).submitBlocks(
+            preconditionBlockNumber,
+            preconditionBlockHash,
+            blocks);
```

Comment:
Why pass this into the protocol and not just do the necessary checks here in the owner contract?

Reply:
We can do that as well